docs/interfaces/auth0clientoptions.html

```diff
@@ -1141,9 +1167,28 @@ <h3><span class="tsd-flag ts-flagOptional">Optional</span> prompt</h3>
 				<aside class="tsd-sources">
 					<p>Inherited from BaseLoginOptions.prompt</p>
 					<ul>
-						<li>Defined in <a href="https://github.com/auth0/auth0-spa-js/blob/fd3f1cb/src/global.ts#L6">src/global.ts:6</a></li>
+						<li>Defined in <a href="https://github.com/auth0/auth0-spa-js/blob/339808f/src/global.ts#L6">src/global.ts:6</a></li>
+					</ul>
+				</aside>
+			</section>
+			<section class="tsd-panel tsd-member tsd-kind-property tsd-parent-kind-interface tsd-is-not-exported">
+				<a name="redirect_uri" class="tsd-anchor"></a>
+				<h3><span class="tsd-flag ts-flagOptional">Optional</span> redirect_<wbr>uri</h3>
+				<div class="tsd-signature tsd-kind-icon">redirect_<wbr>uri<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div>
+				<aside class="tsd-sources">
+					<ul>
+						<li>Defined in <a href="https://github.com/auth0/auth0-spa-js/blob/339808f/src/global.ts#L46">src/global.ts:46</a></li>
 					</ul>
 				</aside>
+				<div class="tsd-comment tsd-typography">
+					<div class="lead">
+						<p>The default URL where Auth0 will redirect your browser to with
+							the authentication result. It must be whitelisted in
+							the &quot;Allowed Callback URLs&quot; in your Auth0 Application&#39;s
```

Comment:
Add "field"

> the &quot;Allowed Callback URLs&quot; field in your ...

Reply:
this is auto generated, but I fixed in the right file as well